src/selectors/getChildren.ts

```diff
@@ -46,13 +47,20 @@ export const hasChildren = (state: State, context: Context) => {
 /** Gets all visible children within a context. */
 export const getChildren = getVisibleThoughts(getAllChildren)
 
-/** Gets all visible children within a context sorted by rank or sort preference. */
+/** Gets all children within a context sorted by rank or sort preference. */
 export const getChildrenSorted = (state: State, context: Context) => {
```

Comment:
Great! I like the way you factored out the visible predicate so that we can use it both ways.

There was an inversion of semantics that occurred that I would like to correct though. Currently, `getChildren*` returns only visible children by default, and `getAllChildren*` includes hidden children. This was intentional, since the most common case is to get only visible children. In your refactor, you have inverted this: the unprefixed functions include hidden children, and functions prefixed with `getVisible` do not.

Let's change this back to the original semantics, specifically:
- Rename `getChildrenSorted` to `getAllChildrenSorted`
- Rename `getVisibleChildrenSorted` to `getChildrenSorted`
- Rename `getVisibleChildrenWithCursorCheck` to `getChildrenWithCursorCheck`

Reply:
Sure. This makes sense. Thanks!